in.varb.Ausweiskopie.yaml

```diff
@@ -0,0 +1,75 @@
+app-id: "in.varb.Ausweiskopie"
+runtime: "org.freedesktop.Platform"
+runtime-version: "24.08"
+sdk: "org.freedesktop.Sdk"
+command: ausweiskopie
+finish-args:
+  - --socket=x11
+  - --share=ipc
```

Comment:
add device=dri if this is gui.

Reply:
Okay... I am not sure if Tk even makes use of GPU acceleration, but I can add it.

Reply:
I didn't know it doesn't support GPU acceleration, try stracing to see what nodes it is accessing in /dev.

Reply:
Just running `strace python3 -m ausweiskopie 2>&1 | grep /dev` outside of flatpak results in nothing. Is this the correct approach?

Reply:
I guess it's not needed then.

Reply:
It's Tk. It's straight from the 90s.